CHANGELOG-SYCL.md

```diff
@@ -0,0 +1,12 @@
+# SYCL CUTLASS Changelog
+
+## [RELEASE NAME GOES HERE](https://github.com/codeplay/cutlass-fork/releases/tag/v???) (2025-04-30)
+
```

Comment:
```suggestion
- Support for Intel GPU data Centre Max (1100  and 1550) 
- Support for Intel Arc B580 Battlemage 
```

Reply:
We haven't tested B570. But it only differs in number of Xe cores and memory. I think it should be reasonable safe to assume that it'll work too.